src/test/java/de/adesso/kicker/notification/NotificationControllerTest.java

```diff
@@ -0,0 +1,106 @@
+package de.adesso.kicker.notification;
+
+import de.adesso.kicker.notification.controller.NotificationController;
+import de.adesso.kicker.notification.exception.NotificationNotExistingException;
+import de.adesso.kicker.notification.exception.WrongReceiverException;
+import de.adesso.kicker.notification.message.MessageDummy;
+import de.adesso.kicker.notification.service.NotificationService;
+import de.adesso.kicker.user.service.UserService;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.BDDMockito.willThrow;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@WebMvcTest(value = NotificationController.class, secure = false)
+public class NotificationControllerTest {
+
+    @MockBean
+    NotificationService notificationService;
+
+    @MockBean
+    UserService userService;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    @DisplayName("A notification should be accepted")
+    @WithMockUser
+    void acceptValidNotification() throws Exception {
+        // given
+        var notification = MessageDummy.messageDeclined();
+
+        // when
+        var result = this.mockMvc.perform(get("/notifications/accept/" + notification.getNotificationId()));
+
+        // then
+        then(notificationService).should(times(1)).acceptNotification(notification.getNotificationId());
```

Comment:
_times(1)_ is not needed it's set to 1 by default

Reply:
Is a change needed?
This is often used like this in other controller tests.

Reply:
I learned this some days ago, I'll fix all others. It's ok